examples/find_and_avoid/worlds/small_world.wbt

```diff
@@ -1,35 +1,18 @@
-#VRML_SIM R2020a utf8
+#VRML_SIM R2021a utf8
 WorldInfo {
-}
-DEF robot E-puck {
-  hidden position_0_0 -9.108562689118283
-  hidden position_0_1 3.1547223230903496
-  hidden linearVelocity_0 -0.0017720631548026364 -1.563979260766478e-07 -0.0007657853543421064
-  hidden angularVelocity_0 0.0016422397371241711 0.2193649305098755 -0.003670303291332604
-  hidden rotation_1 -1 0 0 3.4578079252408895
-  hidden linearVelocity_1 -0.006989572042876959 -2.4592753491424944e-07 -0.0030217810657323318
-  hidden angularVelocity_1 -0.16179636422138877 0.21948214863385254 0.37421916975809205
-  hidden rotation_2 -1 0 0 3.1547223230903496
-  hidden linearVelocity_2 0.0034796935544542658 -8.95102422848352e-08 0.0015108522435370061
-  hidden angularVelocity_2 0.086416191223866 0.21937714440779785 -0.19910698039111127
-  translation -0.41315245128769285 -6.390043543542598e-05 0.3547243076966861
-  rotation -0.0006457489220301296 0.9999993119382065 -0.0009793525634786888 -1.975800098380752
-  name "robot"
-  controller "robot"
-  emitter_channel 0
-  receiver_channel 0
+  coordinateSystem "NUE"
 }
 WoodenBox {
   translation -0.23 0.15 0.15
   size 0.1 0.3 0.7
 }
 DEF target Ball {
-  translation 0.38 0.06991244453845845 -0.37
+  translation 0.38 0.06991368534122255 -0.37
   radius 0.07
 }
 Viewpoint {
```

Comment:
Maybe we can move viewpoint node to the 2nd position.
There are some warning message when the viewpoint node not be included at position 2.

Reply:
Could you please elaborate this. I am not sure what's the problem

Reply:
@ManosMagnus I teseted the code on webots R2021a.
The following is the warning message.
![warningfor](https://user-images.githubusercontent.com/49781698/120415555-4ed0a100-c38e-11eb-9e27-6baa9cc9bc19.PNG)
As you can see, the viewpoint node is currently at the position 5.
![pos](https://user-images.githubusercontent.com/49781698/120415772-ad961a80-c38e-11eb-95a1-1b5a89da8467.PNG)
We can move `Viewpoint{}` to position 2 in this small_world.wbt, or cut and past this node on webots GUI.

Reply:
Fixed!